storage-operator/pkg/controller/volume/volume_controller.go

```diff
@@ -453,32 +453,6 @@ func (self *ReconcileVolume) removePvFinalizer(
 	return self.client.Update(ctx, pv)
```

Comment:
Commit message:

```diff
- This will simplify `deployVolume` which already handle the job polling
+ This will simplify `deployVolume` which already handles the job polling
```

Reply:
Done